.prettierignore

```diff
@@ -1,2 +1,3 @@
 # package.json is formatted by package managers, so we ignore it here
-package.json
\ No newline at end of file
+package.json
+src/codec
```

Comment:
We actually format the codec files in CosmJS so they look consistent and are nicer to read. (We don't lint them obviously.)

Reply:
Ah, that is nice. I will try that.